### PR TITLE
Feature/update hash id to be indexed

### PR DESCRIPTION
**Pull request by xtornasol512**
Enhance the search thought web browser, updating to db index the field "Hash id"

### CODE REVIEW

**Comment by xtornasol512** [comment]
Requires the merging of the  previous PR

File: rexchain/blockchain/behaviors.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ class IOBlockchainize(models.Model):
     signature = models.TextField("Signature with PK and data transaction ", blank=True, default="")
 
     # For tracking Inputs and Outputs
```
Comment (ebarojas) [inline rexchain/blockchain/behaviors.py:32]: 👍

**Review by ebarojas — APPROVED** [review]
Love it. Short and sweet.

🚀

**Comment by ebarojas** [comment]
QA Done, performed migrations in development and created many prescriptions, everything seems to work fine.

**Pull request merged**